R/util.R

```diff
@@ -30,12 +30,22 @@ SetApiKey = function(key) {
   Sys.setenv(API_KEY = key)
 }
 
-# helper function to ensure that dcids input is either
-# a single element list or
-# a multi-element vector
-# to make sure Reticulate converts the input to a Python list
+#' Unset your API Key
```

Comment:
could you elaborate more here? perhaps include that this is for testing purposes, though i'd like to see if there's a way to avoid doing this explicitly, eg. can you call SetApiKey again with an empty string? or include this function there in the test harness that's used to clean up after each test.

Reply:
Actually -- yes. I was hoping to implement a unset API key function in Python that I could use instead of assigning it a pseudo key. But sounds like you'd prefer just using an empty string. I'm good with that.

Reply:
correct